requirements-dev.txt

```diff
@@ -7,7 +7,7 @@ mypy>=0.990,<1.1.0; platform_machine != "aarch64" and python_version >= "3.7"
 black==22.3.0
 types-requests==2.28.11.16
 # Prompting libraries needed for testing
-langchain>=0.0.144,<0.1; python_version>="3.9"
+langchain>=0.0.144,<0.0.174; python_version>="3.9"
```

Comment:
Lowered the `langchain` pin because there seems to be an issue with their `sqlalchemy` usage on Python 3.11 since their v0.0.174 (released three days ago). We can revisit this in a bit, I expect they'll fix this soon.